tools/setup_perf_cluster.sh

```diff
@@ -124,10 +125,15 @@ function get_vm_ip() {
   echo "+++ VM Ip is $VM_IP - visit (http on port 443 is not a typo:) $VM_URL"
 }
 
+function generate_istio_yaml() {
+  Execute sh -c "helm template --namespace $ISTIO_SYSTEM_NAMESPACE --set sidecarInjectorWebhook.enabled=false --set global.proxy.image=proxy   --values install/kubernetes/helm/istio/values-istio-auth.yaml install/kubernetes/helm/istio > install/kubernetes/istio-auth.yaml"
```

Comment:
Why here using `global.proxy.image=proxy` but not `global.proxy.image=proxyv2`? Can you add some comments here to clarify?

Reply:
because this is the v1 version, trying to compare 0.8 with 0.7 for users that would upgrade, without other changes, the rules in the script don't work with v2, will do a v2 version on master